### DISCUSSION TITLE
blankviewer map

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#2964] [Q&A] [answered]
Hi,

I am trying to display backgroundmap in blankviewer, but the map does not appear

![image](https://user-images.githubusercontent.com/53996840/147806100-c293ac25-05ac-4437-8cee-de1100fde3bf.png)

![image](https://user-images.githubusercontent.com/53996840/147805995-ce8bf344-8198-4ff7-a2c6-bef27472230c.png)

**Comment by kckst8** [comment]
What version of the @itwin/viewer-react package are you using?

**Reply by Rahul-0108** [reply]
@kckst8   

The package version : "@itwin/web-viewer-react": "^1.0.8",

**Reply by Rahul-0108** [reply]
Any Solution for this map issue ??

**Reply by kckst8** [reply]
It looks as though this will need to be exposed as a pre-rendering option. I've added this [issue](https://github.com/iTwin/viewer/issues/93)

**Comment by aruniverse** [comment, marked as answer]
This feature has been added as of @itwin/web-viewer-react v2.0.3

**Reply by Rahul-0108** [reply]
still the Map is not visible 
![image](https://user-images.githubusercontent.com/53996840/152433821-83e2c31b-ad91-453e-8190-ed22b913fa9b.png)

![image](https://user-images.githubusercontent.com/53996840/152433648-c2abda9f-cc6a-4224-8fa2-96b1380c7422.png)

**Reply by aruniverse** [reply]
You need to provide mapLayersOptions to the viewer, [see](https://github.com/iTwin/viewer/blob/3434494ed6b86dda02ca6bd48646ec35a8b0f672/packages/modules/viewer-react/src/types.ts?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L111)

**Reply by pmconne** [reply]
@kckst8 v2.0.3 is using 3.0 versions of @itwin packages, correct? As of 3.0, applications are required to supply their own access keys for services like Bing Maps.
Does the viewer provide a way for apps to supply IModelAppOptions.mapLayerOptions and/or TileAdmin.Props.cesiumIonKey?
EDIT: answered by @aruniverse - thanks.

**Reply by kckst8** [reply]
@pmconne correct
